Index: xterm-242/XTerm.ad
===================================================================
--- xterm-242.orig/XTerm.ad
+++ xterm-242/XTerm.ad
@@ -3,7 +3,7 @@
 *saveLines: 1024
 
 *SimpleMenu*BackingStore: NotUseful
-*SimpleMenu*menuLabel.font: -adobe-helvetica-bold-r-normal--*-120-*-*-*-*-iso8859-*
+*SimpleMenu*menuLabel.font: -adobe-helvetica-bold-r-normal--*-120-*-*-*-*-iso*-*
 *SimpleMenu*menuLabel.vertSpace: 	100
 *SimpleMenu*HorizontalMargins: 16
 *SimpleMenu*Sme.height:	16
