--- aclocal.m4
+++ aclocal.m4	2011-11-17 09:31:16.663145702 +0000
@@ -621,10 +621,10 @@ AC_CACHE_CHECK(for $cf_test_message func
 cf_save_LIBS="$LIBS"
 cf_cv_lib_tgetent=no
 if test "$cf_full_tgetent" = yes ; then
-	cf_TERMLIB="otermcap termcap termlib ncurses curses"
+	cf_TERMLIB="otermcap termcap termlib tinfo ncurses curses"
 	cf_TERMTST="buffer[[0]] == 0"
 else
-	cf_TERMLIB="termlib ncurses curses"
+	cf_TERMLIB="termlib tinfo ncurses curses"
 	cf_TERMTST="0"
 fi
 for cf_termlib in '' $cf_TERMLIB ; do
--- configure
+++ configure	2011-11-17 09:47:29.795646131 +0000
@@ -5600,10 +5600,10 @@ else
 cf_save_LIBS="$LIBS"
 cf_cv_lib_tgetent=no
 if test "$cf_full_tgetent" = yes ; then
-	cf_TERMLIB="otermcap termcap termlib ncurses curses"
+	cf_TERMLIB="otermcap termcap termlib tinfo ncurses curses"
 	cf_TERMTST="buffer[0] == 0"
 else
-	cf_TERMLIB="termlib ncurses curses"
+	cf_TERMLIB="termlib tinfo ncurses curses"
 	cf_TERMTST="0"
 fi
 for cf_termlib in '' $cf_TERMLIB ; do
